api/dataset.go

```diff
@@ -871,6 +871,38 @@ func (api *DatasetAPI) getMetadata(w http.ResponseWriter, r *http.Request) {
 	log.Debug("get metadata relevant to version", log.Data{"dataset_id": datasetID, "edition": edition, "version": version})
 }
 
+func (api *DatasetAPI) deleteDataset(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	datasetID := vars["id"]
+
+	currentDataset, err := api.dataStore.Backend.GetDataset(datasetID)
+	if err == errs.ErrDatasetNotFound {
+		w.WriteHeader(http.StatusNoContent) // idempotent
```

Comment:
Missing log line here? Why no content over not found?

Reply:
Added a log line 👍